akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/PathDirectivesSpec.scala

```diff
@@ -304,6 +304,10 @@ class PathDirectivesSpec extends RoutingSpec with Inside {
       "support the flatMap modifier in accept [/yes]" in test("yes")
       "support the flatMap modifier reject [/blub]" in test()
     }
+    {
+      val test = testFor(path(RemainingDecoded.map(_.split(' ').toList)) { echoComplete })
+      "support the map modifier in accept [/yes%20no]" in test("List(yes, no)")
+    }
```

Comment:
Good, that's enough I think - just to see it also handles other things than space 👍